microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1753,6 +2132,35 @@ Receiver:
 
 ---
 
+
+<!-- ====================================================================== -->
+## Managing SmartScreen
+
+<!-- todo: move this section into the above outline structure.  if this is h2, add a "Managing SmartScreen" row to the nav table at top. -->
```

Comment:
@novac42 @peiche-jessica -- Where does the RelNotes 112 section "Managing SmartScreen" fit into page structure?  Move section to there.
See headings-outline at top of PR's main page:
https://github.com/MicrosoftDocs/edge-developer/pull/2510

Reply:
removed v112 items from this PR